content/altinn-studio/reference/monitoring/_index.nb.md

```diff
@@ -5,31 +5,45 @@ description: Altinn App metrikker, telemetri og logger er tilgjengelig i Azure A
 weight: 70
 ---
 
+{{% notice info %}}
+Vi migrerer fra Application Insights SDK til å bruke [OpenTelemetry (OTel)](https://opentelemetry.io/) som en leverandøruavhengig 
+løsning for instrumenterting og eksportering av telemetri fra apper. Dette gjør oss i stand til å kunne tilby eksisterende Application
+Insights løsning, samtidig som utviklere we i stand til å eksportere til egendefinerte monitoreringsløsninger og leverandører, samt forbedre instrumentasjon og fleksibilitet.
+Det betyr også at disse dokumentasjonssidene er under aktiv utbedring og kan være midlertidig ufullstendige.
+{{% /notice %}}
+
 {{% notice info %}}
 Konfigurering av egendefinerte regler og varsler er for øyeblikket ikke tilgjengelig for tjenesteeiere,
-men vi har som mål å støtte dette i løpet av våren 2023.
+men vi har som mål å støtte dette i løpet av høsten 2024.
 {{% /notice %}}
 
-Azure Application Insights (AI) er en utvidelse av
-[Azure Monitor](https://learn.microsoft.com/en-us/azure/azure-monitor/overview) og er det verktøyet vi
-bruker i Altinn for å tilby overvåkingsfunksjonalitet for apper.
+Se [brukerveiledning for instrumentering og monitorering](/nb/altinn-studio/guides/monitor-and-instrument/) for å komme i gang i din app.
+
+Denne dokumentasjonen inneholder nødvendig informasjon for å støtte apputviklere og tjenesteeiere i å
+drifte, overvåke/observere og instrumentere applikasjoner på Altinn 3 plattformen.
 
-AI kan gi deg som apputvikler verdifull innsikt i helse, ytelse og bruk av appen din.
-Med sanntidsovervåking og ytelsesanalyse kan utviklere identifisere og løse problemer før de påvirker
-brukeropplevelsen til en sluttbruker. Feilsøking og varsling* gjør AI til en verdifull ressurs også i drift.
+Dette inkluderer
 
-![Illustrasjon av AI-grafer](ai-overview.png "Illustrasjon av AI-grafer")
+* [Hvordan apper instrumenteres](/nb/altinn-studio/reference/monitoring/instrumentation)
```

Comment:
dette blir vel bare en punktliste av undersidene som må vedlikeholdes manuelt. Kan man heller utbedre description for undersidene og la den listen (se screenshot) snakke for seg selv? 

![image](https://github.com/Altinn/altinn-studio-docs/assets/47737608/57f406d0-550f-4b35-ae44-c9c48a9eb29a)

Reply:
Hmm usikker, tenkte at det kanskje ikke var 1:1 relasjon mellom punkt og side her, men kanskje det er like greit. Tanken var at det kunne være en sånn shortlist over de viktigste seksjonene på tvers av hele dokumentasjonen, mens selve referanse-sidene er mer e2e info om en topic